lib/rubocop/cop/style/file_null.rb

```diff
@@ -42,7 +47,19 @@ class FileNull < Base
         REGEXP = %r{\A(/dev/null|NUL:?)\z}i.freeze
         MSG = 'Use `File::NULL` instead of `%<source>s`.'
 
+        def on_new_investigation
+          return unless (ast = processed_source.ast)
+
+          @contain_dev_null_string_in_file = ast.each_descendant(:str).any? do |str|
+            content = str.str_content
+
+            !invalid_string?(content) && content.downcase == '/dev/null' # rubocop:disable Style/FileNull
```

Comment:
I'm curious why `#downcase` is required here. I don't think it would cause any problems, but perhaps there should be a dedicated test?

Reply:
It is necessary to pass the existing test.
https://github.com/rubocop/rubocop/blob/v1.69.1/spec/rubocop/cop/style/file_null_spec.rb#L53-L54

Reply:
Ah, I see, thanks.